tests/fixtures/course.json

```diff
@@ -1,366 +1,366 @@
 {
-	"conclude": {
```

Comment:
Seems there's some pretty aggressive tabs -> spaces going on in this JSON file. Tabs are fine here, please revert this change. Only convert between tabs and spaces when they are mixed.